New synthvivor loadouts & Fiorina spelling fixes

JohnDoe50974 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request
Adds three new synth survivor loadouts to the pool the game randomly selects from upon spawning - the firefighter, the UA military synthvivor with an unique dogtag variant and carrying misc low-grade spare ammo and medicine, and a PMC support synthetic with PMC ID and IFF. Also fixes code that misspells "Fiorina" as "FLorina"
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The synthvivor loadout randomizer system has been received warmly, so further expansion on it is welcome.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl: 50RemAndCounting
add: Added a few new synth survivor loadouts that are picked randomly upon spawning as one.
spellcheck: fixed all occurences in code and ingame where Fiorina is spelled as FLorina
fix: fixed synthetics still having the CO headset as their replacement headset in the vendor.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

FoxyStalin commented:
Real PMC Support synth hell yes.
Look forward to that one.

Geevies commented on code/modules/gear_presets/_select_equipment.dm:
```diff
@@ -774,6 +774,49 @@ var/list/rebel_rifles = list(
+			H.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel, WEAR_BACK)
+			H.equip_to_slot_or_del(new /obj/item/storage/belt/medical/full, WEAR_WAIST)
+			H.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health, WEAR_EYES)
+			faction = FACTION_PMC
```
this change's the datum's faction, not the synthvivor. if this does properly work on the synthvivor (VV to confirm), then you'll just need to reset the faction to its initial value at the start of this proc so the other synthvivors (futureproofing) doesn't get the wrong faction

JohnDoe50974 replied:
Yeah I set the faction to the default survivor faction group on the next case immediately after to fix this

Geevies replied:
wuh

JohnDoe50974 commented:
Removed the codefix from https://github.com/cmss13-devs/cmss13/pull/977 and added it here instead. Will need a Fix tag on this PR and said tag removed from the linked PR.
